src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/TextController.cs

```diff
@@ -0,0 +1,155 @@
+using System;
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Helpers;
+using Altinn.Platform.Storage.Interface.Models;
+using Altinn.Platform.Storage.Repository;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace Altinn.Platform.Storage.Controllers
+{
+    [ApiController]
+    [Route("storage/api/v1/applications/{org}/{app}/texts")]
+    public class TextController : ControllerBase
+    {
+        private readonly ITextRepository _textRepository;
+        private readonly ILogger _logger;
+
+        /// <summary>
+        /// Initalizes a new instance of the <see csref="TextController.cs"/> class
+        /// </summary>
+        /// <param name="textRepository">the text repository handler</param>
+        /// <param name="logger">dependency injection of logger</param>
+        public TextController(ITextRepository textRepository, ILogger<TextController> logger)
+        {
+            _textRepository = textRepository;
+            _logger = logger;
+        }
+
+        [HttpPost]
```

Comment:
What's the reasoning behind not specifying the language id in the route for this method?

Reply:
I was unsure of that one. But the main idea is that the language prop is an inherent property of the text resource, so it felt natural to me to not specify it "twice". I have noe strong opinions either way, what do you think?

Reply:
I was mostly curious. Seems fair. Resolving the conversation.